Assets/Scripts/UIManager.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class UIManager : MonoBehaviour
+{
+    public static UIManager Instance { get; private set; }
+
+    [SerializeField]
+    private GameObject silencePanel = null;
+
+    [SerializeField]
+    private GameObject infoPanel = null;
+
+    private List<Snitch> snitches = new List<Snitch>();
+    private Snitch selectedSnitch = null;
```

Comment:
selectedSnitch is assigned but not used. Is this intended?

Reply:
Yeah, this is just set up for selection later, but I realized I was probably jumping the gun on this specific ticket